tc.rb

```diff
@@ -43,13 +43,6 @@ def db_name
       :data_required => 'yes'
     ),
 
-    # The data warehouse importer needs read access to the tc DB.
-    'db access'.with(
-      :db_name => db_name,
-      :username => 'dw.theconversation.edu.au',
-      :check_table => 'content'
-    ),
```

Comment:
@nullobject while removing this bit, I noticed that further up is:

```ruby
def db_name
  YAML.load_file(app_root / 'config/database.yml')[env.to_s]['database']
end
```

.. just something to be aware of before we provision the new boxes

Reply:
Ah good point, that definitely won't work.

Reply:
I added a card to revisit: https://trello.com/c/zT6sxJXK/9351-new-servers-fix-app-deps-to-handle-database-urls